src/main/java/nl/tudelft/model/LevelFactory.java

```diff
@@ -89,11 +90,42 @@ public Level getLevel(int id) throws IllegalArgumentException {
                 return getLevel3();
             case 4:
                 return getLevel4();
+            case 5: 
```

Comment:
Maybe the boss level should be the 2nd level so Maiko can test it more easily.

< review